extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js

```diff
@@ -113,6 +113,10 @@ RdfSchemaAlignmentDialog.prototype._constructBody = function(body) {
                     }
                 }
             );
+        var fileType = window.prompt("filetype","");
+        $.post("command/rdf-extension/save-rdf-skeleton",{model: JSON.stringify(self.getJSON()), projectId: theProject.id, fileType: fileType},function(data){
```

Comment:
Maybe we can make this a bit more readable with newlines and indentation, the oneliner could be hard to read.